checksums.sh

```diff
@@ -15,18 +15,19 @@ then
 fi
 
 SUBJECT_FOLDER=$1
+QSUB_FILE=".qsub.out.*"
 
-echo "###########################################"
-echo "############ Checksum of docker image #####"
-echo "###########################################"
-
-grep "Digest: sha256:" .qsub.out.*
+count=`ls -a 2>/dev/null | grep $QSUB_FILE | wc -l`
+if [ $count != 0 ]
+then
```

Comment:
You could just do `ls ${QSUB_FILE} && grep "Digest ...`
ls $QSUB_FILE will fail if there is no file matching the pattern and `&&` won't execute the grep. Question: should we stop the script if the file is not there, i.e., the container checksum can't be obtained?